src/earn/saga.test.ts

```diff
@@ -416,18 +419,21 @@ describe('depositSubmitSaga', () => {
 })
 
 describe('withdrawSubmitSaga', () => {
-  const mockRewards = [{ tokenId: mockArbArbTokenId, amount: '1' }]
+  const mockRewards = [
+    { tokenId: 'arbitrum-sepolia:0x912ce59144191c1204e64559fe8253a0e49e6548', amount: '0.01' },
+  ]
+  const mockPool = mockRewardsPositions[0] as EarnPosition
   const serializableWithdrawTx: SerializableTransactionRequest = {
     from: '0xa',
     to: '0xb',
-    value: '100',
+    value: '1075',
```

Comment:
Where does the 1075 come from? It looks like the mock pool has 6 decimals so a little confused what math is being done.